jykuo-love-shiritori/twp
# Oauth2

Resolves #46 

### Changes


TODO Listed in issue

### Testing Approach (if unconventional)


- Create a user in the DB (form validation in `/signup` isn't implemented)
- Go to `/login` and click on the button
- Enter your login credentials
- After being redirected to the home page you can navigate to `/APITest` and you should see "pong" on the screen
- Clicking on "Logout" inside the drop down in the top right corner would redirect you to the login page
- Navigating to any path inside the `<ProtectedBoundary />` component would also redirect you to the login page

### Additional context


N/A
